### ISSUE TITLE
🐞 Bug (Accessibility): Theme toggle button is inaccessible and lacks a tooltip

### ISSUE THREAD

**Issue by K9Patel** [Rajput-xv/Health-Report-Analyzer#122]
### Describe the bug

The theme toggle button is inaccessible and confusing.

1.  It's invisible to screen readers, so visually impaired users don't know what the button is for.

2.  It lacks a tooltip, so users who hover over it with a mouse don't get any confirmation of what it does.

<img width="132" height="101" alt="Image" src="https://github.com/user-attachments/assets/0a89afc8-3d03-4aa3-b62b-e4f4cd24dc14" />

i want to make like to enhance the user experience instead of this simple button
